snippets/saved_replies.md

```diff
@@ -24,9 +24,11 @@ It's not always feasible, or even possible, to fill out the XXX used in the snip
 
 Thanks for your report. It looks as if this is actually a question about usage and not development because XXX
 
-To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users), which you can also search. If you prefer more interactive help, join or our [IRC channel](https://webchat.freenode.net/?randomnick=1&channels=prometheus), #prometheus on irc.freenode.net. Please be aware that our IRC channel has no logs, is not searchable, and that people might not answer quickly if they are busy or asleep. If in doubt, you should choose the mailing list.
+To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users).
 
-Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+If you haven't looked already you might find the answer in the [official docs and examples](https://prometheus.io/docs/introduction/overview/) as well as searching in the google [users](https://groups.google.com/forum/#!forum/prometheus-users) or [devs](https://groups.google.com/forum/#!forum/prometheus-developers) groups or ask  in the #prometheus [IRC channel ](https://webchat.freenode.org/?channels=prometheus), and I am sure there will be someone to help you out.
```

Comment:
I don't see a need to mention that these are google groups - they're mailing lists. You've some extra spaces too.